discovery/db_discovery.go

```diff
@@ -38,11 +38,15 @@ type DBOrchestratorPoolCache struct {
 	orchBlacklist         []string
 }
 
-func NewDBOrchestratorPoolCache(ctx context.Context, node *core.LivepeerNode, rm common.RoundsManager, orchBlacklist []string) (*DBOrchestratorPoolCache, error) {
+func NewDBOrchestratorPoolCache(ctx context.Context, node *core.LivepeerNode, rm common.RoundsManager, orchBlacklist []string, discoveryTimeout int) (*DBOrchestratorPoolCache, error) {
 	if node.Eth == nil {
 		return nil, fmt.Errorf("could not create DBOrchestratorPoolCache: LivepeerEthClient is nil")
 	}
 
+	if discoveryTimeout > 0 {
+		getOrchestratorsCutoffTimeout = time.Duration(discoveryTimeout) * time.Millisecond
```

Comment:
We shouldn't rely on a static global field if we are sending the argument on a specific struct constructor. Could you add a `getOrchestratorsCutoffTimeout` field to both `DBOrchestratorPoolCache` and `orchestratorPool` and remove that static field instead to avoid any confusion?